pkg/agent/agentserver/server.go

```diff
@@ -117,31 +117,41 @@ func (s *ProxyServer) serveRecvFrontend(stream agent.ProxyService_ProxyServer, r
 			}
 
 			if err := s.Backend.Send(pkt); err != nil {
-				klog.Warningf("send packet to Backend failed: $v", err)
+				klog.Warningf("send packet to Backend failed: %v", err)
 			}
 			s.PendingDial[pkt.GetDialRequest().Random] = &ProxyClientConnection{
 				Mode:      "grpc",
 				Grpc:      stream,
 				connected: make(chan struct{}),
 			}
 			klog.Info("DIAL_REQ sent to backend") // got this. but backend didn't receive anything.
-		case agent.PacketType_DIAL_RSP:
-			klog.Info("received DIAL_RSP;; ignore on Server")
-		case agent.PacketType_ACK:
-			klog.Info("received ACK")
-		case agent.PacketType_CLOSE:
-			klog.Info("received CLOSE")
+
+		case agent.PacketType_CLOSE_REQ:
+			klog.Infof("received CLOSE_REQ(id=%d)", pkt.GetCloseRequest().ConnectID)
```

Comment:
Tempted to suggest we up the log level of close. It seems like it should be higher than normal data packets. Maybe just file an issue to review our log levels.